sample/codecov.yml

```diff
@@ -1,4 +1,4 @@
-# Sample CodeCov config for private repository
+# For the private repository, set up a TeamBot https://docs.codecov.com/docs/team-bot
 codecov:
   bot: luongvo
```

Comment:
Can we replace it with something else, set a personal name here is quite weird for a public repo? 🤔

Reply:
@markgravity updated in https://github.com/nimblehq/flutter-templates/pull/238/commits/44e679c8ed299c06c17b17dd8f7266ab1300b2e4